src/Api/UrlGeneratorInterface.php

```diff
@@ -31,6 +31,11 @@
  */
 interface UrlGeneratorInterface
 {
+    /**
+     * Allow to generate url using the globally configured strategy.
+     */
+    public const DEFAULT = -1;
```

Comment:
Maybe would it be more consistent zither the rest of the API to use null instead? Not really sure, just asking.

Reply:
That would work and simplify the the conditional statements indeed as pointed in your other comment.
the only downside is that I'll have to silence PHPStan errors because the [IriConverterInterface](https://github.com/api-platform/core/blob/90f422faa56beb3afa51aea75053c5a9333886dc/src/Api/IriConverterInterface.php#L45) and the [ContextBuilderInterface](https://github.com/api-platform/core/blob/90f422faa56beb3afa51aea75053c5a9333886dc/src/JsonLd/ContextBuilderInterface.php) expects int and will be passed `null`

Do you think it is worth it ?

Reply:
And we can't use `0` I'd be to keep `-1`!

Reply:
We must respect the contract of the interface.

Reply:
Anyway, `UrlGeneratorInterface::DEFAULT` is not a good name for a constant. I'd say it should be `DEFAULT_STRATEGY`?